### PR TITLE
#718 update docs for is empty render

### PR DESCRIPTION
**Pull request by jordann-nulogy**
Updated docs in `Shallow` and `ReactWrapper` to add `isEmptyRender()`

### CODE REVIEW

File: docs/api/ReactWrapper/isEmptyRender.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# `.isEmptyRender() => Boolean`
+
+Returns true if the component rendered nothing, i.e., null or false.
```
Comment (ljharb) [inline docs/api/ReactWrapper/isEmptyRender.md]: let's surround null/false in backticks, throughout

**Comment by aweary** [comment]
ping @jordann-nulogy! can you address the feedback and fix the merge conflicts?

File: docs/api/ReactWrapper/isEmptyRender.md (hunk 1 above)
Comment (neville1355) [inline docs/api/ReactWrapper/isEmptyRender.md]: Sorry I didn't see this before. I've updated all references of true, null and false with b ackticks.

Comment (jordann-nulogy) [inline docs/api/ReactWrapper/isEmptyRender.md]: Sorry I didn't see this before. I've updated all references of true, null and false with backticks.

**Comment by jordann-nulogy** [comment]
@aweary thanks for the ping, I didn't realize this was blocked. Must have missed it in my inbox.

File: docs/README.md [hunk 2]
```diff
@@ -47,6 +47,8 @@
     * [key()](/docs/api/ShallowWrapper/key.md)
     * [last()](/docs/api/ShallowWrapper/last.md)
     * [map(fn)](/docs/api/ShallowWrapper/map.md)
+    * [isEmpty()](/docs/api/ShallowWrapper/isEmpty.md)
```
Comment (nfcampos) [inline docs/README.md]: isEmpty was already on this list, look at line 46

File: docs/README.md [hunk 3]
```diff
@@ -101,6 +103,8 @@
     * [key()](/docs/api/ReactWrapper/key.md)
     * [last()](/docs/api/ReactWrapper/last.md)
     * [map(fn)](/docs/api/ReactWrapper/map.md)
+    * [isEmpty()](/docs/api/ReactWrapper/isEmpty.md)
```
Comment (nfcampos) [inline docs/README.md]: isEmpty is already present on line 102

**Comment by ljharb** [comment]
This was taken care of in 80a74b4.

**Pull request closed without merging**